cmake/recipes/h5pp.cmake

```diff
@@ -8,8 +8,9 @@ endif()
 message(STATUS "Third-party: creating target 'h5pp'")
 
 SET(H5PP_ENABLE_EIGEN3 ON)
-SET(H5PP_ENABLE_FMT ON)
+SET(H5PP_ENABLE_FMT OFF)
```

Comment:
if this is not required anymore, can we drop the fmt cmake recipe and include?

Reply:
fmt is theoretically required downstream - it would be good to keep the recipe in the wildmeshing repo perhaps? I got the habit of manually packaging fmt when i used spdlog from before fmt::join was in not in the fmt version packaged with spdlog - using the spdlog packaged version of fmt always felt a bit awkward in general :(.